### PR TITLE
Feature/dummy classifier learner

### PR DESCRIPTION
**Pull request by Lguyogiro**
* Added sklearn's `DummyClassifier` (used for generating a simple baseline) to the list of supported classifiers.
* Updated Documentation accordingly
* Added a test that covers each of the three possible 'strategy' values ("stratified", "most_frequent", and "constant").

### CODE REVIEW

**Comment by desilinguist** [comment]
LGTM 👍 but let's merge this after the encrypted slack credentials one since I want to get that into `master` ASAP.

**Comment by desilinguist** [comment]
Hmm, I thought I fixed the Slack integration to show approvals but I guess not.

File: tests/test_classification.py [hunk 1]
```diff
@@ -210,6 +210,38 @@ def check_sparse_predict_sampler(use_feature_hashing=False):
...
+    )
+
+    # Ensure predictions are correct for all strategies.
+    correct = []
+    for model_args, expected_output in toy_data:
+        learner = Learner('DummyClassifier', model_kwargs=model_args)
+        learner.train(train_fs)
+        predictions = learner.predict(test_fs)
+        correct.append(np.array_equal(expected_output, predictions))
+    assert_equal(correct, [True, True, True])
```
Comment (desilinguist) [inline tests/test_classification.py]: I think the already imported `eq_` would work fine for this no? Do we really need `assert_equal` here?

Comment (Lguyogiro) [inline tests/test_classification.py]: No we don't. I've updated the test to use `eq_` and removed the `assert_equal` import

**Comment by desilinguist** [comment]
@aoifecahill if you can approve this, I'll merge it in.

**Pull request merged**